lib/stoplight/light/runnable.rb

```diff
@@ -26,6 +27,21 @@ def run
         end
       end
 
+      def lock(color)
+        raise Error::IncorrectColor unless Color::LOCKABLE_COLORS.include?(color)
+
+        state = case color
+                when Color::RED then State::LOCKED_RED
+                when Color::GREEN then State::LOCKED_GREEN
+                end
```

Comment:
What do you think? 

```suggestion       
        state = case color
                when Color::RED then State::LOCKED_RED
                when Color::GREEN then State::LOCKED_GREEN
                else raise Error::IncorrectColor
                end
```

1. you don't need the `LOCKABLE_COLORS` constant anymore
2. You don't have to synchronize the case statement with the `LOCKABLE_COLORS` constant

Reply:
I think it's cool. I'll implement it right away :)